pkg/clustermanager/cluster_manager.go

```diff
@@ -226,6 +234,23 @@ func (c *ClusterManager) CreateWorkloadCluster(ctx context.Context, managementCl
 		return nil, fmt.Errorf("error generating workload kubeconfig: %v", err)
 	}
 
+	logger.V(3).Info("Run provider specific create operations")
+	err = c.Retrier.Retry(
+		func() error {
+			err = provider.RunSpecificCreateOps(ctx, clusterSpec, workloadCluster)
```

Comment:
you could just do `return provider.RunSpecificCreateOps(ctx, clusterSpec, workloadCluster)`